lib/matobj.gi

```diff
@@ -293,6 +297,10 @@ InstallMethod( Matrix,
 
 InstallMethod( Matrix,
     [ IsSemiring, IsList ],
+# rank higher than a method in the semigroups package, which otherwise jumps
+# in and causes an error when testing
+# line 318 of semigroups-3.4.0/gap/elements/semiringmat.gi
+20,
```

Comment:
I think we'll have to sort it out together with the Semigroups folks, though, don't want to break that package (paging @james-d-mitchell).